src/views/landing.njk

```diff
@@ -0,0 +1,83 @@
+{% extends "layout.njk" %}
+{% extends "govuk/template.njk" %}
```

Comment:
I bet we do not need that since `layout.njk` extends `govuk/template.njk` as well.

Reply:
Well spotted